app/scrapers/twitter.py

```diff
@@ -0,0 +1,27 @@
+import urllib.request
```

Comment:
The other scrapers are using Requests https://github.com/fossasia/query-server/blob/master/app/scrapers/generalized.py#L2  Doing so will save you a ton of aggravation.